apis/Google.Cloud.Logging.V2/Google.Cloud.Logging.Log4Net/MemoryLogQueue.cs

```diff
@@ -68,42 +69,23 @@ public DateTimeRange EnqueueAsync(IEnumerable<LogEntryExtra> logEntries)
                     }
                     var sizedEntry = new LogEntryExtraSize(entry, _maxMemorySize > 0 ? entry.Entry.CalculateSize() : 0);
                     _q.Enqueue(sizedEntry);
-                    if (entry.Id > _maxIdSeen)
```

Comment:
The queue used to have to maintain the maximum seen ID, but this is no longer necessary as it's now done in LogUploader. This means the queueing logic is slightly simplified, which will help a bit if/when writing other local queues (e.g. file-based).